GitUI/CommandsDialogs/SettingsDialog/Pages/FormBrowseRepoSettingsPage.cs

```diff
@@ -77,5 +106,10 @@ private void cboTerminal_Enter(object sender, EventArgs e)
         {
             _cboTerminalPreviousIndex = cboTerminal.SelectedIndex;
         }
+
+        private void chkProcessHistoryAsTab_InfoClicked(object sender, EventArgs e)
+        {
+            OsShellUtil.OpenUrlInDefaultBrowser(_processHistoryUrl);
```

Comment:
A little surprising this tooltip icon that is also a clickable help link.
I only think about clicking while seeing this code.
I see that we have that elsewhere but here the cursor change to the "hand" is missing...